MANIFEST.in

```diff
@@ -5,5 +5,5 @@ graft visualizer
 graft examples
 prune docs/build
 recursive-include *.py
-include README.rst LICENSE CONTRIBUTORS.rst HISTORY.rst tox.ini Makefile
+include README.rst LICENSE CONTRIBUTORS.rst HISTORY.rst tox.ini Makefile src/priority/py.typed
```

Comment:
Shouldn't `py.typed` already be covered by the `graft priority` statement above?
But I actually suspect that this should be `graft src/priority` instead?

Reply:
I don't know, but this is how I've seen it done elsewhere

Reply:
I'm still trying to fully understand all the moving parts, but from https://packaging.python.org/guides/using-manifest-in/ I would argue that this is not necessary, because we already have `py.typed` in the `package_data`.